plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -95,13 +95,14 @@ public static function convert_line( $p_text ) {
 	 *
 	 * @param string $line The Markdown syntax to parse
 	 * @access protected
-	 * @return void if markdown starts with # symbol followed by any character | string html representation generated from markdown.
+	 * @return null|string HTML representation generated from markdown or null if
+                      markdown starts with # symbol immediately followed by 
+                      any character other than space.
```

Comment:
> I don't think it is correct, because if the regex doesn't match (e.g. if the raw text is # blah at beginning of the line, then the parsedown parent method won't be called.

I'm not sure what you meant for? The logic is just simple:

`if raw text is like # blah (#+space+text) then the parent::blockHeader() will be called`.